src/tribler-core/tribler_core/components/simple_storage/simple_storage.py

```diff
@@ -0,0 +1,70 @@
+import asyncio
+import logging
+
+from pydantic import BaseModel
+
+from tribler_core.utilities.path_util import Path
+
+DEFAULT_SAVE_INTERVAL = 5 * 60  # force Storage to save data every 5 minutes
+
+
+class StorageData(BaseModel):
+    last_processed_torrent_id: int = 0
+
+
+class SimpleStorage:
+    """ SimpleStorage is object storage that stores data in JSON format and uses
+    `pydantic` `BaseModel` for defining models.
+
+    It stores data on a shutdown and every 5 minutes.
+
+    limitations:
+     *   No transactions: last five-minute changes can be lost on Tribler crash, so the
+            application code should be tolerable to this and be ready, for example, to
+            process the same torrents again after the Tribler restart.
+     *   If two instances of the application try to use the same storage simultaneously,
+            they will not see the changes made by another instance.
+    """
+
+    def __init__(self, path: Path, save_interval: float = DEFAULT_SAVE_INTERVAL):
+        """
+        Args:
+            path: path to the file with storage. Could be a path to a non existent file.
+            save_interval: interval in seconds in which the storage will store a data to a disk.
+        """
+        self.logger = logging.getLogger(self.__class__.__name__)
+        self.data = StorageData()
+
+        self.path = path
+        self.save_interval = save_interval
+
+        self._loop = asyncio.get_event_loop()
+        self._task: asyncio.TimerHandle = self._loop.call_later(self.save_interval, self._save_and_schedule_next)
+
+    def load(self):
+        """ Load data from `self.path`. In case the file doesn't exist, the function
+        will create the data with defaults values.
+        """
+        self.logger.info(f'Loading storage from {self.path}')
+        try:
+            self.data = StorageData.parse_file(self.path)
+        except Exception:  # pylint: disable=broad-except
+            self.logger.info('The storage file does not exist or corrupted. Create a new storage.')
```

Comment:
Shouldn't it be a warning at least?

Reply:
It is not always a warning. Sometimes it is just a message (for example in the case that file doesn't exist).

Reply:
But I've added the requested change:
```python
            self.logger.warning(e)

```